File: tests/test_remove_vowels.py
import unittest
from remove_vowels import remove_vowels

class TestRemoveVowels(unittest.TestCase):
    def test_remove_vowels(self):
        self.assertEqual(remove_vowels("hello"), "hll")
        self.assertEqual(remove_vowels("AEIOUaeiou"), "")
        self.assertEqual(remove_vowels("Python Programming"), "Pythn Prgrmmng")
        self.assertEqual(remove_vowels(""), "")

if __name__ == "__main__":
    unittest.main()

File: tests/test_remove_vowels.rs
#[cfg(test)]
mod tests {
    use super::super::strings::remove_vowels;

    #[test]
    fn test_remove_vowels() {
        assert_eq!(remove_vowels("hello"), "hll");
        assert_eq!(remove_vowels("AEIOUaeiou"), "");
        assert_eq!(remove_vowels("Python Programming"), "Pythn Prgrmmng");
        assert_eq!(remove_vowels(""), "");
    }
}
